src/Adapter/MSTest.CoreAdapter/Resources/Resource.resx

```diff
@@ -285,7 +285,7 @@
     <value>The test method {0}.{1} has multiple attributes derived from ExpectedExceptionBaseAttribute defined on it. Only one such attribute is allowed.</value>
   </data>
   <data name="LegacyScenariosNotSupportedWarning" xml:space="preserve">
-    <value>Warning : A testsettings file or a runsettings with a ForcedLegacyMode set to true is not supported with the MSTest V2 Adapter.</value>
+    <value>Warning : A vsmdi file, testsettings file or a runsettings file with a ForcedLegacyMode set to true is not supported with the MSTest V2 Adapter.</value>
```

Comment:
How about change it to "A testsettings file, a runsettings file with a ForcedLegacyMode set to true or a vsmdi file is not supported with the MSTest V2 Adapter." ? As vsmdi is less popular. /cc @pvlakshm

Reply:
Yes, that sounds better.

Reply:
Done.